packages/react/src/analytics/generated.ts

```diff
@@ -0,0 +1,71 @@
+/*
+ * This file is autogenerated, do NOT edit
+ */
+export type FathomRegisteredEventName =
+    | "TX_ERC20_APPROVAL"
+    | "TX_KPI_TOKEN_CREATION"
+    | "TX_KPI_TOKEN_REDEMPTION"
+    | "TX_ORACLE_FINALIZATION"
+    | "TX_CUSTOM";
+
+export interface FathomSiteEvent {
+    id: string;
+    object: "event";
+    name: FathomRegisteredEventName;
+    currency: string | null;
+    created_at: string;
+}
+
+export interface FathomSiteInformation {
+    siteId: string;
+    events: FathomSiteEvent[];
+    timestamp: number;
+}
+
+export const siteEvents: FathomSiteInformation = {
+    siteId: "YPUQOWIV",
```

Comment:
This should probably be an env

Reply:
Fixed in 43032df6185e2fc20d89b2de720e23e45d81d7f6.